### DISCUSSION TITLE
FOR THE MODS ONLY!

### DISCUSSION THREAD

**Discussion by SakuraKitsunee** [CRobbins0867/ProdigyHacking--MobileHacking#10] [General]
1. what we need to do

- [] get more mods 
- [x] make a github page!

**Comment by irhrhd** [comment]
How many more?

**Reply by SakuraKitsunee** [reply]
probably around 1 or 2

**Comment by SakuraKitsunee** [comment]
@irhrhd we need to shout out our respiratory somehow, do you have any ideas?

**Reply by irhrhd** [reply]
Yes in the Big  Prodigy hacking 1

**Reply by SakuraKitsunee** [reply]
no we cant shout out there they wont let us

**Reply by irhrhd** [reply]
Oh then idk

**Reply by irhrhd** [reply]
Let me think

**Reply by irhrhd** [reply]
https://github.com/irhrhd/Prodigy-Hacking-Mobile/blob/main/Steps

**Comment by SakuraKitsunee** [comment]
@irhrhd whats your email im making a real website so i need it for contacts

**Reply by SakuraKitsunee** [reply]
??

**Reply by irhrhd** [reply]
360684@students.bethelsd.org
Whats the name of the website

**Reply by irhrhd** [reply]
I can literally just give you a website

**Reply by irhrhd** [reply]
Heelooooooo you gonna answerrrrrr

**Reply by SakuraKitsunee** [reply]
is at the about section
